types/mithril/stream/index.d.ts

```diff
@@ -0,0 +1,44 @@
+declare namespace Stream {
+	export type Combiner<T> = (...streams: any[]) => T;
```

Comment:
Move this inline into `stream.combine`'s type. Usually, people will specify this as a anonymous function, so an inline type makes sense. Plus, that's what autocomplete often does already.